t/proxyunits.lua

```diff
@@ -552,6 +552,15 @@ function mcp_config_routes(zones)
         return zones.dead(r)
     end
 
+   pfx_get["deadrespcode"] = function(r)
+       local res = zones.dead(r)
+
+       if res:code() == mcp.MCMC_CODE_SERVER_ERROR then
+         return "CODE_CORRECT\r\n"
```

Comment:
Small nitpick: In most of these tests I try to use valid protocol in responses. This currently works, but if we ever add strict return code parsing in the test client or server it would fail and the tests would need to be updated.

You can add `ERROR ` in front of both of these `CODE_` lines: `ERROR code_correct\r\n` etc.